_sass/modules/_nav.scss

```diff
@@ -104,13 +104,23 @@
 
 // Expandy nav items
 .arrow-r::before {
-  content: "+";
-  margin-right: 8px;
+  content: '';
+  position: absolute;
+  background: url('../img/icon-caret-right.svg') no-repeat center/cover;
+  width: 24px;
```

Comment:
As I remember we have variable for this icon size. Use it , please.

Reply:
Done